docs/sql-performance-tuning.md

```diff
@@ -298,6 +319,14 @@ AQE converts sort-merge join to broadcast hash join when the runtime statistics
        </td>
        <td>3.2.0</td>
      </tr>
+     <tr>
+       <td><code>spark.sql.adaptive.localShuffleReader.enabled</code></td>
```

Comment:
This is a non-internal configuration which has been defined and used for a while.

https://github.com/apache/spark/blob/786594734bd79017ebd42eb117b62958afad07bb/sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala#L702-L709